htmlgenerator/base.py

```diff
@@ -213,8 +213,8 @@ class If(BaseElement):
     def __init__(
         self,
         condition: typing.Union[bool, Lazy],
-        true_child: BaseElement,
-        false_child: typing.Optional[BaseElement] = None,
+        true_child: typing.Any,
+        false_child: typing.Any = None,
```

Comment:
Should ```false_child``` not be ```typing.Optional```?

Reply:
I think `typing.Optional` is a subtype of `typing.Any`

Reply:
proof by IDE: 
![image](https://user-images.githubusercontent.com/11404846/144015433-0f037d86-c46c-4914-8d9e-6f5ea324d4b2.png)

Reply:
Might be. My thought was that Optional[Any] would be more expressiv. It makes clear that the ```If``` can be used without an "else" block. Docs regarding Optional also suggest this type of use (see https://docs.python.org/3/library/typing.html#typing.Optional).